packages/edit-site/src/components/sidebar-edit-mode/template-card/style.scss

```diff
@@ -57,6 +57,10 @@
 			min-width: auto;
 		}
 	}
+
+	& .block-editor-block-patterns-list__item .block-editor-block-preview__container {
+		border: 1px solid $gray-300;
```

Comment:
What would be a good mechanism to highlight patterns when they are not hovered/selected? I found that if the pattern has a white background it does not stand out of its own in the list.

Reply:
How about the `box-shadow: inset 0 0 0 1px rgba(0, 0, 0, 0.2);` that we use in other similar situations (like the color indicators).

Reply:
We might consider making a var for that, if we're going to start using it in different places.